english/polar-bear-attack/polar-bear-attack.md~

```diff
@@ -0,0 +1,7 @@
+You leap and jump on the back of the polar bear. It bucks and tries to throw
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.